geneocr-api/1.0/Dockerfile

```diff
@@ -0,0 +1,58 @@
+# base image
+FROM python:3.10-slim
+
+# metadata
+LABEL base_image="python:3.10-slim"
+LABEL version="1"
+LABEL software="GeneOCR API"
+LABEL software.version="1.0.0"
+LABEL about.summary="API of geneocr tool, which reads gene names in images"
+LABEL about.home="https://github.com/bedapub/geneocr-api"
+LABEL maintainer="Jitao David Zhang"
+LABEL maintainer.email="jitao_david.zhang@roche.com"
+
+# install dependencies
+RUN export DEBIAN_FRONTEND=noninteractive && \
+    apt-get update && \
+    apt-get -y --no-install-recommends install \
+    build-essential \
+    libpcre++-dev \
+    gfortran \
+    ca-certificates \
+    cmake \
+    git \
+    wget 
+#    python3 \
+#    python3-pip \
+#    zlib1g-dev \
+#    libpng-dev \
+#    libfile-slurp-perl && \
+#    apt-get autoclean && rm -rf /var/lib/apt/lists/*
+
+RUN git clone --depth=1 https://github.com/bedapub/geneocr-ui.git && \
+    cd geneocr-ui && \
+    ls
+
+WORKDIR /code
+
+COPY ./app/requirements.txt /code/requirements.txt
```

Comment:
COPY does not work here, this is not a local file, but I suppose it is in what you clone , I suppose you expect a RUN cp ....